processing/src/main/java/org/apache/druid/math/expr/ConstantExpr.java

```diff
@@ -145,6 +152,18 @@ protected ExprEval<T> realEval()
       return eval;
     }
 
+    @Override
+    public String stringify()
+    {
+      return origExpr.stringify();
```

Comment:
Is `stringify` in all possible `origExpr` thread-safe? This approach would require that. Would be good to have a comment here mentioning that too.

Alternatively is it possible to implement this using the ExprEval? Like `eval.toExpr.stringify`? Would generate more garbage, but be simpler structurally and then we wouldn't have to worry about any possible issues related to retaining a ref to the original expr. Also, stringify isn't called in hot loops so the extra garbage isn't as important.